README.md

```diff
@@ -64,10 +33,13 @@ def top_level_task():
 
 **More FlexFlow PyTorch examples**: see the [pytorch examples folder](https://github.com/flexflow/FlexFlow/tree/master/examples/python/pytorch).
 
+## TensorFlow Keras Support
+Users can use FlexFlow to accelerate the training procedure of existing TensorFlow Keras models, please read the [Keras Support](https://flexflow.readthedocs.io/en/latest/keras.html).
```

Comment:
```suggestion
FlexFlow prioritizes PyTorch compatibility, but also includes frontends for [Tensorflow Keras](https://flexflow.readthedocs.io/en/latest/keras.html) and [ONNX](...) models.
```

Reply:
Also, @gabrieleoliaro correct me if I'm wrong here but shouldn't this be a local md link that RTD converts into a full url?

Reply:
yes, basically, if you put the link to the corresponding `.md` file here, RTD will automatically convert that into an URL when building the docs :)

Reply:
the link should be a local file?

Reply:
Yes, it should link to the corresponding md file in this repo (the RTD is automatically generated from the contents of this repo)